runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/RPASections/RPABackground/RPABackground.module.scss

```diff
@@ -0,0 +1,62 @@
+@import "#src-landing/scss/main.scss";
+
+$ARROWS_HEIGHT: 450px;
+$MIN_CONTENT_WIDTH: 500px;
+
+@mixin backgroundArrowImage($image-url, $position) {
+  align-self: center;
+  width: 100%;
+  height: $ARROWS_HEIGHT;
+  background-image: url($image-url);
+  background-position: center $position;
+  background-repeat: no-repeat;
+  mix-blend-mode: soft-light;
+  opacity: 1;
+  z-index: 1;
```

Comment:
since you are using z index in this component use somewhere `isolation` rule https://developer.mozilla.org/en-US/docs/Web/CSS/isolation

Reply:
thanks for the link!

Reply:
done